api/v1alpha1/application_types.go

```diff
@@ -184,17 +186,49 @@ const (
 	PENDING     StatusNames = "Pending"
 )
 
+func MarshalledReplicas(replicas interface{}) *apiextensionsv1.JSON {
+	replicasJson := &apiextensionsv1.JSON{}
+	var err error
+
+	replicasJson.Raw, err = json.Marshal(replicas)
+	if err == nil {
+		return replicasJson
+	}
+
+	return nil
+}
+
+func GetReplicasFloat(jsonReplicas *apiextensionsv1.JSON) (float64, error) {
+	var result float64
+	err := json.Unmarshal(jsonReplicas.Raw, &result)
+
+	return result, err
+}
+
+func GetReplicasStruct(jsonReplicas *apiextensionsv1.JSON) (Replicas, error) {
+	result := Replicas{
+		Min:                  2,
+		Max:                  5,
+		TargetCpuUtilization: 80,
+	}
```

Comment:
kunne denne Replicas structen vært en default Const struct, siden du bruker den to ganger?

Reply:
Seems you can't use a const, but I was thinking of something simple like:
```
var DefaultReplicas = Replicas{
	Min:                  2,
	Max:                  5,
	TargetCpuUtilization: 80,
}
```

And then just using that struct instead of defining the default twice :)

Reply:
Fine by me!